index.html

```diff
@@ -51,7 +67,25 @@
                 </div>
             </div>
         </header>
-
+        <section class="section-carousel">
+            <div class="column-wrapper">
+                <div class="carousel-slide">
+                    <div class="carousel-slide__text-container">
+                        <h1 class="heading_level-1">Мужская и женская одежда с символикой HeadHunter</h1>
+                        <button>Перейти в каталог</button>
+                    </div>
+                    <div class="carousel-slide__image-contaier">
+                        <img class="carousel-slide__image" src="images/carousel.png" alt="SlideImage"/>
+                    </div>
+                    <div class="carousel-slide__pages-contaier">
+                        <span class="carousel-page__round carousel-page__active"></span>
```

Comment:
снова неправильное использование/именования.
Если ты назвал элемент `carousel-page__...`, то он **обязательно** должен находить внутри блока `carousel-page`.
И с названием модификатора снова проблемы. Модификатор `carousel-page__active` относиться к элементу `carousel-page__round`, его название должно строиться по схеме `carousel-page__round_НАЗВАНИЕ-МОДИФИКАТОРА`
Например, вот так можно сделать
```suggestion
                    <div class="carousel-page">
                        <span class="carousel-page__round carousel-page__round_active"></span>
```